packages/rum-core/src/domain/rumEventsCollection/action/actionCollection.ts

```diff
@@ -98,9 +98,9 @@ function processAction(
     },
     autoActionProperties
   )
-  const inForeground = foregroundContexts.isInForegroundAt(action.startClocks.relative)
-  if (inForeground !== undefined) {
-    actionEvent.view = { in_foreground: inForeground }
+  const inActivePageState = pageStateHistory.isActiveAt(action.startClocks.relative)
+  if (inActivePageState !== undefined) {
+    actionEvent.view = { in_foreground: inActivePageState }
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**❗ warning:** ‏ ‏FMU, `inActivePageState` can't be `undefined`, there is maybe a missing test here
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->

Reply:
Indeed! I checked and this comes from when there was a feature flag. I'll remove this check